keras/layers/recurrent.py

```diff
@@ -109,7 +109,8 @@ class Recurrent(Layer):
             else a symbolic loop will be used.
             Unrolling can speed-up a RNN,
             although it tends to be more memory-intensive.
-            Unrolling is only suitable for short sequences.
+            Unrolling is only suitable for short sequences
+            and for sequences with a length greater than 1.
```

Comment:
"and for sequences with a length greater than 1." is more confusing than anything. Please remove.